src/main/java/org/gaul/s3proxy/S3ProxyHandlerJetty.java

```diff
@@ -153,6 +158,7 @@ public void handle(String target, Request baseRequest,
                 throw throwable;
             }
         }
+        accessLogger.logResponse(target, baseRequest, request, response);
```

Comment:
This doesn't execute on any exception paths because there are several `return` statements.